sqlup-mode.el

```diff
@@ -59,6 +59,12 @@
 (require 'cl-lib)
 (require 'sql)
 
+
+;; Optionally load the redis package for the keywords
+(defvar redis-keywords)
```

Comment:
I'm uncomfortable defining a variable name that doesn't belong to me. Can we do something different here, like define a `sqlup-redis-keywords` variable and wrap this logic in a function that gets called later? Or will that not fix the compiler error?